docs/pt/docs/how-to/graphql.md

```diff
@@ -0,0 +1,62 @@
+# GraphQL
+
+Como **FastAPI** é baseado no padrão **ASGI**, é muito fácil integrar qualquer biblioteca **GraphQL** também compatível com ASGI.
+
+Você pode combinar *operações de caminho* normais do FastAPI com GraphQL na mesma aplicação.
```

Comment:
although "operações de caminho" is also used, the most common translation in the docs is "operações de rota". I think it's nice to keep de standard 😃